extensions/agent/src/dialogs/pickScheduleDialog.ts

```diff
@@ -81,6 +70,17 @@ export class PickScheduleDialog {
 
 			this.loadingComponent = view.modelBuilder.loadingComponent().withItem(formModel).component();
 			this.loadingComponent.loading = true;
+			this.model.initialize().then((result) => {
```

Comment:
You're inside an async function here, could just use await.

Reply:
This is intentional. You don't want to block the UI thread.